extensions/wikia/ContentReview/controllers/templates/ContentReviewSpecialController_index.php

```diff
@@ -26,7 +26,7 @@
 				<td><?= htmlspecialchars( $review['wiki'] ) ?></td>
 				<td><a href="<?= Sanitizer::encodeAttribute( $review['url'] ) ?>" target="_blank"><?= htmlspecialchars( $review['title'] ) ?></a></td>
```

Comment:
No reason to decide for user to open link in new tab. Especially that opening in same window doesn't break anything.